src/python/pants/testutil/BUILD

```diff
@@ -1,22 +1,25 @@
 # Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+# The `pantsbuild.pants.testutil` distribution is built as one version for all the various Python
+# versions supported. However the build script used to build the `pantsbuild.pants` wheels runs for
+# each separate Python version to produce version specific wheels, so we need to cancel the default
+# interpreter constraints set by that process, using the `setup_py.python_requires` field, and also
+# tagging the wheel for all the supported Python versions.
 python_distribution(
     name="testutil_wheel",
     dependencies=[":testutil", ":pants_integration_test", ":rule_runner", ":py_typed"],
-    # This python wheel tag must align with the interpreter constraints.
     wheel_config_settings={"--global-option": ["--python-tag", "py37.py38.py39"]},
```

Comment:
Now that we populate `Requires-Python` metadata properly, I actually think we can drop the wheel config setting altogether. As rough supporting evidence think about or browse wheels out on PyPI and note that many have Requires-Python of something more specific than ">3", e.g.: ">=3.6", and yet the published wheel is `project-version-py3-none-any.whl`.

Reply:
For example: https://pypi.org/project/tomli/#files